HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -974,52 +974,85 @@ stores this on the given stream
 method:: printOn
 prints this on the given stream
 
+
 subsection:: random
+Unless noted otherwise, when the receiver and arguments to these methods are all link::Classes/Integer::s, the return will be as well.
```

Comment:
maybe better: 
Unless noted otherwise, the methods return numbers of the same class as the receiver and arguments.

Currently, SimpleNumber has only Floats and Interger as subclasses, but this may be extended.

Reply:
your wording is ambiguous—this line is intended to indicate that you get an int back if and only if all participants are ints. "of the same class" is unspecific when the receiver and arguments are of different types.

Reply:
I see. Then you should probably write "only when" or add "only" to the end.

Reply:
done. also added a note that sum3rand is always a Float.